### PR TITLE
New version: Liberica JDK18-Full 18.0.0.37

### PR DESCRIPTION
**Pull request by PakhomGolynga**
- [x ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/56086)

### CODE REVIEW

File: manifests/b/BellSoft/LibericaJDK/18/Full/18.0.0.37/BellSoft.LibericaJDK.18.Full.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+PackageIdentifier: BellSoft.LibericaJDK.18.Full
+PackageVersion: 18.0.0.37
+MinimumOSVersion: 6.1.0.0
+InstallerType: msi
```
Comment (Trenly) [inline manifests/b/BellSoft/LibericaJDK/18/Full/18.0.0.37/BellSoft.LibericaJDK.18.Full.installer.yaml:4]:
```suggestion
InstallerType: wix
```

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/b/BellSoft/LibericaJDK/18/Full/18.0.0.37/BellSoft.LibericaJDK.18.Full.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  InstallerSha256: 0DADA529AB7BB152BF8E41CF6A6F9BDE18B35CE892C43DB01B455E04619A9152
+  ProductCode: '{DC7820D7-096E-48B6-9EA7-527981AE119F}'
+- Architecture: x86
+  InstallerUrl: https://download.bell-sw.com/java/18+37/bellsoft-jdk18+37-windows-i586-full.msi
+  InstallerSha256: A976E4B2B218FF6C50A00103FC8EB9B41EDC2A61E6A69D4A1DC69385DBB05DA4
+  ProductCode: '{682C17C3-097F-479E-9C36-B25EBCE5773E}'
+- Architecture: arm64
+  InstallerUrl: https://download.bell-sw.com/java/18+37/bellsoft-jdk18+37-windows-aarch64-full.msi
+  InstallerSha256: 337D4AF06D87216E811002553B9078AF75BAE140ECE34DD9B2D68FBC04A6FDC2
+  ProductCode: '{82C18FCD-9E5D-4D22-97E8-77D24FC97C08}'
```
Comment (ItzLevvie) [inline manifests/b/BellSoft/LibericaJDK/18/Full/18.0.0.37/BellSoft.LibericaJDK.18.Full.installer.yaml:19]:
```suggestion
  ProductCode: '{B7BBDAAF-E471-4265-A96E-F068ED80F868}'
```

File: manifests/b/BellSoft/LibericaJDK/18/Full/18.0.0.37/BellSoft.LibericaJDK.18.Full.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+PackageVersion: 18.0.0.37
+MinimumOSVersion: 6.1.0.0
+InstallerType: wix
+Scope: machine
+ReleaseDate: 2022-03-22
+Installers:
+- Architecture: x64
+  InstallerUrl: https://download.bell-sw.com/java/18+37/bellsoft-jdk18+37-windows-amd64-full.msi
+  InstallerSha256: 0DADA529AB7BB152BF8E41CF6A6F9BDE18B35CE892C43DB01B455E04619A9152
+  ProductCode: '{DC7820D7-096E-48B6-9EA7-527981AE119F}'
```
Comment (ItzLevvie) [inline manifests/b/BellSoft/LibericaJDK/18/Full/18.0.0.37/BellSoft.LibericaJDK.18.Full.installer.yaml:11]:
```suggestion
  ProductCode: '{6BC5FE82-3B5E-40B5-A923-C9453630E667}'
```

File: manifests/b/BellSoft/LibericaJDK/18/Full/18.0.0.37/BellSoft.LibericaJDK.18.Full.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+- Architecture: x86
+  InstallerUrl: https://download.bell-sw.com/java/18+37/bellsoft-jdk18+37-windows-i586-full.msi
+  InstallerSha256: A976E4B2B218FF6C50A00103FC8EB9B41EDC2A61E6A69D4A1DC69385DBB05DA4
+  ProductCode: '{682C17C3-097F-479E-9C36-B25EBCE5773E}'
```
Comment (ItzLevvie) [inline manifests/b/BellSoft/LibericaJDK/18/Full/18.0.0.37/BellSoft.LibericaJDK.18.Full.installer.yaml:15]:
```suggestion
  ProductCode: '{1ED4EE5D-5CD5-428A-B17C-7D0E4DBD59EB}'
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
ProductCodes need to be fixe

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**